src/redux/imageMetadata.js

```diff
@@ -0,0 +1,42 @@
+import produce from 'immer';
+import {
+  getImageMetadata,
+  saveImageMetadata,
+} from '../handlers/localstorage/globalSettings';
+
+// // -- Constants --------------------------------------- //
+const MERGE = 'imageMetadata/MERGE';
+
+export const imageMetadataCacheLoadState = () => async dispatch => {
+  const metadataCache = await getImageMetadata();
+  dispatch({
```

Comment:
Since this dispatch doesn't have an id passed into it, the MERGE action will set the draft to look like draft[undefined] = blablabla. I've made a PR here: https://github.com/rainbow-me/rainbow/pull/832